docs/docusaurus.config.js

```diff
@@ -0,0 +1,160 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+    title: 'FastTrackML',
+    tagline: 'Experiment tracking server focused on speed and scalability',
+    favicon: 'favicon.ico',
+    onBrokenLinks: 'throw',
+    onBrokenMarkdownLinks: 'warn',
+
+    url: 'https://G-Research.github.io',
+    baseUrl: '/fasttrackml/',
```

Comment:
I think the URL will be https://fasttrackml.io/
@c-rindi could you please confirm?

Reply:
Yes, I guess that is the one.